static/mixins/polling.jsx

```diff
@@ -6,11 +6,82 @@ var PollingMixin = {
       pollingActive: true
     };
   },
+  pushNotification(){
+    let body;
+    let {app, env, number} = this.props.params;
+    var path = `/deploys/${app}/${env}/${number}`;
 
-  componentWillMount() {
+    let hostname = location.hostname
+    let protocol = location.protocol
+    let port     = location.port
+
+    if(this.state.task == undefined){
+      let {name} = this.state.deploys[0].app
+      let {environment, number} = this.state.deploys[0]
+
+      var path = `/deploys/${name}/${environment}/${number}`
+    }
+
+    if(hostname == "localhost"){
+      var url  = `${protocol}//${hostname}:${port}${path}`
+      var ourl = `${protocol}//${hostname}:${port}${path}`
+    }else{
+      var url  = `${protocol}//${hostname}${path}`
+      var ourl = `${protocol}//${hostname}${path}`
```

Comment:
What's a `ourl` and how does it differ from `url`? Looks the same to me.

Also, the variable name should be more clear. What does the `o` in `ourl` mean? Out?